src/components/pseudonym/AnonymousConsentGate.styles.scss

```
@@ -140,6 +140,11 @@
 	justify-content: center;
 	gap: 6px;
 	padding: 8px 14px;
+	// Both action buttons are pills. This used to be declared per button, and
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)